### PR TITLE
KUZ-682: renamed DSL keywords in code examples

### PR DESCRIPTION
**Pull request by scottinet**
Following https://github.com/kuzzleio/kuzzle/pull/392, this PR updates code examples with the newly renamed DSL keywords

### CODE REVIEW

File: source/includes/_kuzzleDataCollection.md [hunk 1]
```diff
@@ -265,12 +265,12 @@ $filters = [
   'filter' => [
     'and' => [
       [
-        'terms' => [
+        'ub' => [
```
Comment (j33f) [inline source/includes/_kuzzleDataCollection.md]: ub ?

File: source/includes/_kuzzleDataCollection.md [hunk 2]
```diff
@@ -265,12 +265,12 @@ $filters = [
...
           'status' => ['idle', 'wantToHire', 'toHire', 'riding'],
         ]
       ],
       [
-        'terms' => [
+        'ub' => [
```
Comment (j33f) [inline source/includes/_kuzzleDataCollection.md]: ub again ?

Comment (scottinet) [inline source/includes/_kuzzleDataCollection.md]: fixed

File: source/includes/_kuzzleDataCollection.md (hunk 1 above)
Comment (scottinet) [inline source/includes/_kuzzleDataCollection.md]: fixed

**Pull request merged**